oneview/resource_server_profile.go

```diff
@@ -1185,6 +1185,19 @@ func resourceServerProfile() *schema.Resource {
 											},
 										},
 									},
+									"host_name": {
+										MaxItems: 1,
+										Type:     schema.TypeSet,
```

Comment:
please make it to typeList as hostName is just one list.

Reply:
Are we sure that max item is 1 for hostname?

Reply:
Yes because this is suppose to have one block of hostname, thats why maxItems =1
this way we restrict user to not to create mutiple blocks of host_name.